integration-tests/test/rnode.py

```diff
@@ -205,6 +205,12 @@ def __init__(self, *, container: Container, deploy_dir: str, command_timeout: in
     def __repr__(self) -> str:
         return '<Node(name={})>'.format(repr(self.name))
 
+    def get_node_pem_cert(self) -> bytes:
+        return self.shell_out("cat", rnode_certificate).encode('utf8')
+
+    def get_node_pem_key(self) -> bytes:
+        return self.shell_out("cat", rnode_key).encode('utf8')
```

Comment:
How about this instead?
```
from pathlib import Path

file_bytes = Path(rnode_key_path).read_bytes()
file_text = Path(rnode_key_path).read_text()
```
I've changed `rnode_key` to `rnode_key_path` to make it clear that it's path, not key content.

Reply:
The `rnode_key_path` is actually inside the Rnode container which is not mounted to the host. So the direct python read is not working. 

But I agree to change the name to be more clear.

Reply:
Got it. Makes sense.